src/state/nftMarket/helpers.ts

```diff
@@ -59,7 +61,45 @@ export const getNftsMarketData = async (collectionAddress: string): Promise<Nfts
     )
     return res.collection.NFTs
   } catch (error) {
-    console.error('Failed to fetch NFTs', error)
+    console.error('Failed to fetch NFTs from collection', error)
+    return []
+  }
+}
+
+export const getNftsMarketData = async (where = {}): Promise<NftSubgraphEntity[]> => {
```

Comment:
Used to return market data for a users' nfts. 

The where condition is crucial. Examples: 

Market data about tokens in a users' wallet: `{ tokenId_in: [122, 121] }`
Market data about tokens a user has on sale: `{ currentSeller: "0x... }`